### ISSUE TITLE
Request: Please provide a preference to automatically mark articles as read

### ISSUE THREAD

**Issue by ziffusion** [nilsbraden/ttrss-reader-fork#447]
It would be useful to have the application automatically mark articles as read when:

1. Articles scroll past the top.
2. All articles above the one clicked. 

This would be extremely valuable for workflows where folks have to scroll past long lists of articles to find a small number that are actually useful.

**Comment by nilsbraden** [comment]
Actually you can long-press an article and mark all articles above as read. Not the most convenient solution for long lists, I give you that. But it works. :)

**Comment by ziffusion** [comment]
Yes I do realize that that option exists. But having to do it all the time
can be a pain. And then you forget once in a while, and have to find your
place again.

For certain workflows - and I'll bet there are quite a few people out there
- this is a pretty valuable feature. Google Reader used to have this option
BTW.
